docs/configuration/index.md

```diff
@@ -627,6 +627,15 @@ Support for 64-bit floating point columns was released in Druid 0.11.0, so if yo
 |--------|-----------|-------|
 |`druid.indexing.doubleStorage`|Set to "float" to use 32-bit double representation for double columns.|double|
 
+### SQL compatible null handling
+Also prior to version 0.13.0, Druid string columns treated `''` and `null` values as interchangeable, and numeric columns were unable to represent `null` values, coercing `null` to `0`. Druid 0.13.0 introduced an at the time undocumented mode which enabled SQL compatible null handling, allowing string columns to distinguish empty strings from nulls, and numeric columns to contain null rows.
```

Comment:
Nix the "also", people usually aren't reading these straight through, so it's best for each section to stand alone.

Reply:
Nix "at the time undocumented"; no need to mention that in the current notes.

Reply:
done